frontend/src/v5/store/tickets/tickets.helpers.ts

```diff
@@ -15,8 +15,206 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import { formatMessage } from '@/v5/services/intl';
 import { FederationsHooksSelectors } from '@/v5/services/selectorsHooks/federationsSelectors.hooks';
+import { TITLE_INPUT_NAME } from '@/v5/ui/routes/viewer/tickets/ticketsForm/ticketsForm.component';
+import { nullableNumber, requiredNumber, trimmedString } from '@/v5/validation/shared/validators';
+import { isEmpty } from 'lodash';
+import * as Yup from 'yup';
+import { EditableTicket, ITemplate, ITicket, PropertyDefinition } from './tickets.types';
 
 export const modelIsFederation = (modelId: string) => (
 	!!FederationsHooksSelectors.selectContainersByFederationId(modelId).length
 );
+
+export const getTicketDefaultValues = ({ title, properties, modules }: ITicket) => ({
+	title,
+	properties,
+	...modules,
+});
+
+export const filterNonEditablePropertiesFromTemplate = (template) => {
+	const propertyIsEditable = ({ readOnly }) => !readOnly;
+
+	return {
+		...template,
+		properties: (template.properties || []).filter(propertyIsEditable),
+		modules: (template.modules || []).map((module) => ({
+			...module,
+			properties: module.properties.filter(propertyIsEditable),
+		})),
+	};
+};
+
+const templatePropertiesToTicketProperties = (properties = []) => (
+	properties.reduce(
+		(ticketProperties, prop) => ({
+			...ticketProperties,
+			[prop.name]: prop.default,
+		}),
+		{},
+	)
+);
+
+export const getEditableTicketFromTemplate = (template: ITemplate): EditableTicket => {
```

Comment:
No need to specify "fromTemplate", template  is alrady in the the argument. Would change the naming still getDefaultTicket(template). To me, that tells me that its a ticket (based on the template as it is an argument) that has the default properties.